web-ui/view/common.ml

```diff
@@ -86,14 +86,17 @@ let form_for ~x_ref ~action ~csrf_token ~submit_button ~input_value =
           ();
       ])
 
-let form_rebuild_step ~variant ~csrf_token =
+let form_rebuild_step ~variant ~csrf_token ?(show = true) () =
+  let display_none = if show then "" else "display:none" in
```

Comment:
![image](https://user-images.githubusercontent.com/22150236/195367785-9dbb91a8-59a6-4f76-815b-90a08b3fd8f3.png)
The button doesn't seem to be redrawn once the step is finished.

Reply:
I think that in this particular case, since the build has passed, there won't be a Rebuild button. The Rebuild button should only be redrawn for failed builds.